src/collections.ts

```diff
@@ -31,3 +31,15 @@ export function createCollectionResponses(
 ${nameRecordMap}
 }`
 }
+
+export function createTypedPocketbase(
+  collectionNames: Array<string>
+): string {
+  const nameRecordMap = collectionNames
+    .map((name) => `\tcollection(idOrName: '${name}'): RecordService<${toPascalCase(name)}Response>`)
+    .join("\n")
+  return `export type TypedPocketBase = PocketBase & {
+\tcollection(idOrName: string): RecordService
```

Comment:
Do you think this "fallback" should be kept? By having it here the user can pass in collection names that are not generated by `pocketbase-typegen`. That can be seen as both a good and a bad thing, as by having it there is no type checking for invalid collection names.

Reply:
Good point. I would remove it since we have the complete list of collections. An type error is a good hint that they need to regenerate